optuna/study.py

```diff
@@ -880,6 +880,32 @@ def load_study(
     # type: (...) -> Study
     """Load the existing :class:`~optuna.study.Study` that has the specified name.
 
+    Example:
+
+        .. testsetup::
+
+            import os
+            if os.path.exists("example.db"):
+                raise RuntimeError("DB file already exists. Please remove example.db")
```

Comment:
A nit, but please add a blank line between the import line and the code body. And also, I found that we tend to use `database` instead of `DB`.
```suggestion
            import os

            if os.path.exists("example.db"):
                raise RuntimeError("The database file already exists. Please remove example.db.")
```